ts/Stock/StockToolsBindings.ts

```diff
@@ -287,8 +293,9 @@ bindingsUtils.manageIndicators = function (
             series.remove(false);
 
             if (indicatorsWithAxes.indexOf(series.type) >= 0) {
+                const removedYAxisHeight = yAxis.options.height;
                 yAxis.remove(false);
-                navigation.resizeYAxes();
+                navigation.resizeYAxes(removedYAxisHeight as string);
```

Comment:
Instead of telling TS it will be a string, we should rather support both possible values (string and number). Or make sure `height` is cast to string

Reply:
In this case, we are sure that the `height` of the indicator axis that is being removed is always a string as percents. To remove an axis through gui, we need to add it first. When adding, the height is defined here (always as string):
```js
yAxes[index].update({
    height: position.height + '%',
    top: position.top + '%',
...
```

Reply:
Ah, makes sense. Thanks!